### DISCUSSION TITLE
剑指offer题目C++版本贡献

### DISCUSSION THREAD

**Discussion by ChunelFeng** [doocs/leetcode#334] [General] [answered]
你好，我想贡献剑指offer中，C++部分题目的解答，请求如何操作？
方便的话，请联系微信：  ChunelFeng

**Comment by yanglbme** [comment, marked as answer]
可以在每道题增加 Solution.cpp 题解，同时更新 README.md

![image](https://user-images.githubusercontent.com/21008209/105620315-a8e8b500-5e36-11eb-9284-55545cb783a3.png)

**Comment by yanglbme** [comment]
我的微信是 YLB0109，可以通过一下，也可以直接在这里交流。

**Reply by Qiu-IT** [reply]
小弟过来班门弄斧一下，或许日后来有参与Doosc小伙伴来到这会看到，这也许能给给杨哥减轻下负担。因为我第一次提交的时候也因为规范的问题给组织带来许多不便。首先一定要装Git，这个去[**官网**](https://git-scm.com)下载就好了。之后把项目fork到本地并给`git checkout -b 新枝名字` 进行修改。修改的规范要注意的是，第一 代码要格式化 第二要在README.md 那里也要补充你的代码做成代码块。其实就是对照前面贡献者的样版就好。之后就是git常规三步走 `git add a.cpp` `git commit a.cpp -m` `git push origin 新枝名字`（传到你刚刚创建好的新枝）。这个[**地方**](https://github.com/firstcontributions/first-contributions/blob/main/translations/README.zh-cn.md)解释挺详细的，杨哥在Github主页也分享了。愿来到这里的小伙伴一起码个未来。
